src/debug/debugSession.ts

```diff
@@ -262,7 +262,8 @@ export class DebugSession implements IDebugSession {
     private async setupPortForward(podName: string, debugPort: number, appPort?: number): Promise<ProxyResult> {
         kubeChannel.showOutput(`Setting up port forwarding on pod ${podName}...`, "Set up port forwarding");
         const proxyResult = await this.createPortForward(this.kubectl, podName, debugPort, appPort);
-        kubeChannel.showOutput(`Created port-forward ${proxyResult.proxyDebugPort}:${debugPort} ${appPort ? proxyResult.proxyAppPort + ":" + appPort : ""}`);
+        const portStr = appPort ? `${proxyResult.proxyAppPort}:${appPort}` : "";

```

Comment:
`appPortStr` - there are too many ports in play for the unadorned name to be clear.

Reply:
👍